tests/integration/new_relations/test_new_relations.py

```diff
@@ -87,6 +87,26 @@ async def test_database_relation_with_charm_libraries(ops_test: OpsTest, databas
         )
         await ops_test.model.wait_for_idle(apps=APP_NAMES, status="active", raise_on_blocked=True)
 
+        # Check that on juju 3 we have secrets and no username and password in the rel databag
+        if hasattr(ops_test.model, "list_secrets"):
```

Comment:
If we have `list_secrets()` we're on juju 3

Reply:
Why not `JujuVersion.has_secrets`... Just for consistency with the generic concept of checking for Juju3?